coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = '.hs'
```

Comment:
Should be a tuple. I remember I told you once in your previous pr, c.f. https://github.com/coala/coala/pull/5748#discussion_r215634169, but you are making the same mistake again 😅